scripts/make-meta-icons.js

```diff
@@ -0,0 +1,70 @@
+/*
+  Create icons with stroke and m:parameter options for 
+  custom icon attribute styling in our SDKs.
+ */
+const path = require('path');
+const { gatherIcons, write } = require('./utils');
+
+function metaMap({ data }) {
+  data['$'] = {
+    ...data['$'],
+    'xmlns:m': 'https://www.mapbox.com',
+    viewBox: '0 0 23 23',
+    height: '23',
+    width: '23'
+  };
+
+  data.path.forEach((p, index) => {
+    p.$ = {
+      ...p.$,
+      transform: 'translate(4 4)',
+      fill: '#000'
+    };
+
+    // Add a stroked shape to below by unshifting an item into the array
+    data.path.unshift({
```

Comment:
Now stroke paths also have fill parts, so they will be rendered twice. It's not visible as our colors are not transparent, but not optimal.
P.S. Am I right that mapnik doesn't support `paint-order` attribute? If it does, it's better to just put `paint-order: stroke` and not duplicate the entire path.

Reply:
Ahhhh this would be good to confirm. I was sort of mirroring behavior we do today for the maki editor but it might be as simple as using stoke differently. Going to treat that as a followup and merge+publish this for now.